security/pfSense-pkg-snort/files/usr/local/pkg/snort/snort.inc

```diff
@@ -163,15 +152,9 @@ function snort_load_suppress_sigs($snortcfg, $track_by=false) {
 
 	$suppress = array();
 
-	if (!is_array($config['installedpackages']['snortglobal']))
-		return;
-	if (!is_array($config['installedpackages']['snortglobal']['suppress']))
+	if (!config_get_path('installedpackages/snortglobal/suppress/item'))
 		return;
-	if (!is_array($config['installedpackages']['snortglobal']['suppress']['item']))
-		return;
-	$a_suppress = $config['installedpackages']['snortglobal']['suppress']['item'];
-
-	foreach ($a_suppress as $a_id => $alist) {
+	foreach (config_get_path('installedpackages/snortglobal/suppress/item') as $a_id => $alist) {
```

Comment:
Same as previous comment, add a default `[]` so the foreach will always work.

Reply:
Added default array return.